### PR TITLE
Adding inaSpeechSegmenter

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HeavyTony2** [comment]
https://github.com/conda-forge/keras-feedstock/issues/103

**Comment by HeavyTony2** [comment]
https://github.com/conda-forge/keras-feedstock/pull/104

File: recipes/inaSpeechSegmenter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - numpy
+    - pandas
+    - scikit-image
+    - pyannote.core
+    - matplotlib-base
+    - pyro4
+    - pytextgrid
+    - pysoundfile
+    - onnxruntime *=*_cuda   #not avalaible for osx
+    - __linux
```
Comment (HeavyTony2) [inline recipes/inaSpeechSegmenter/meta.yaml:35]: @conda-forge/help-python I'm not sure if this is the right way for a noarch package only avalaible for linux

**Comment by HeavyTony2** [comment]
@conda-forge/help-python ready for review

**Comment by HeavyTony2** [comment]
@conda-forge/help-python Can you confirm if this is an acceptable way to set a noarch package that can only run on linux?
https://github.com/conda-forge/staged-recipes/blob/8ded15628286de1b46700faaa9554c82225173c9/recipes/inaSpeechSegmenter/meta.yaml#L35

File: recipes/inaSpeechSegmenter/meta.yaml (hunk 1 above)
Comment (HeavyTony2) [reply recipes/inaSpeechSegmenter/meta.yaml:35]: @ocefpaf what are the indications?

Comment (ocefpaf) [reply recipes/inaSpeechSegmenter/meta.yaml:35]: Things like:

```
<<<<<<< HEAD
```

but I guess I was looking at an old diff.

Comment (HeavyTony2) [reply recipes/inaSpeechSegmenter/meta.yaml:35]: So is the recipe correct?

**Pull request merged**